### DISCUSSION TITLE
The icon search of the official document is not visible

### DISCUSSION THREAD

**Discussion by w1ndys** [squidfunk/mkdocs-material#7280] [Q&A] [answered]
I don’t know if this is a network problem or a problem with the official backend. I can’t search for any emoticon icons here

![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/a8d535db-5452-447b-b373-330262cb04a0)

![image](https://github.com/squidfunk/mkdocs-material/assets/109416673/515f49b7-0211-47dd-9200-181776eea049)

**Comment by w1ndys** [comment]
This doesn’t seem to be a network problem. I saw that other people have the same problem

https://github.com/squidfunk/mkdocs-material/issues/7278#/

https://github.com/squidfunk/mkdocs-material/issues/7277#/

**Reply by squidfunk** [reply]
Yes, sorry, it's only now that I could set my hands on a computer. I'll fix it immediately.

**Reply by squidfunk** [reply, marked as answer]
It's fixed.

**Reply by w1ndys** [reply]
Thank you so much for your great efforts.
